assets/sass/components/reader-revenue-manager/_googlesitekit-rrm-setup-module.scss

```diff
@@ -0,0 +1,55 @@
+/**
+ * Reader Revenue Manager Setup styles.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-setup-module__publication-create-screen {
+		.googlesitekit-setup-module__title {
+			color: $c-surfaces-on-surface;
+			font-family: $f-primary;
+			font-size: $fs-body-lg;
+			font-weight: $fw-normal;
+			letter-spacing: $ls-m;
+			line-height: $lh-body-lg;
+			margin-bottom: 0;
+		}
+
+		.googlesitekit-setup-module__description {
+			color: $c-surfaces-on-background-variant;
+			font-family: $f-primary;
+			font-size: $fs-body-md;
+			font-weight: $fw-normal;
+			letter-spacing: $ls-s;
+			line-height: $lh-link-md;
+			margin-top: 0;
+		}
+
+		.googlesitekit-setup-module__action {
+			margin-top: 40px;
+
+			.mdc-button {
+				font-family: $f-secondary;
+				font-weight: $fw-medium;
+			}
+
```

Comment:
I realise this matched the design at the time of writing, however:

- The `font-family` was incorrect in Figma. I [raised this with Sigal](https://www.figma.com/design/f2AFGUqfH8rqi9BpJAwhHi?node-id=171-2291#893264858) and she's now fixed the button styling, which should be Google Sans Text as per the standard CTA buttons, so this `font-family` override can be removed.
- We have an existing issue to fix the `font-weight` for buttons to match the design system: https://github.com/google/site-kit-wp/issues/8856. As such, it's probably worth omitting this explicit `font-weight`, the CTA will then be consistent with others and we can fix them all at once via 8856.

```suggestion
```

Reply:
Thanks for addressing this @hussain-t. I'll leave this comment thread open for future reference.